lib/src/iso/bmff/boxes/VisualSampleEntryBox.ts

```diff
@@ -0,0 +1,25 @@
+import type { SampleEntryBox } from './SampleEntryBox.js';
+
+/**
+ * ISO/IEC 14496-15:2014 - 12.1.3.1 avc1/2/3/4, hev1, hvc1, encv
+ *
+ * @group ISOBMFF
+ *
+ * @beta
+ */
+export type VisualSampleEntryBox<T = 'avc1' | 'hev1' | 'hvc1' | 'encv'> = SampleEntryBox & {
```

Comment:
Is it meant to be with `extends` and you have default type

```suggestion
export type VisualSampleEntryBox<T extends 'avc1' | 'hev1' | 'hvc1' | 'encv' = <default_type>> = SampleEntryBox & {
```

or multiple types are intentional?